README.md

```diff
@@ -10,14 +10,14 @@ Add this line to your application's Gemfile:
 
 ```ruby
 group :test, :development do
-  gem 'datarockets-style', '~> 1.5.0'
```

Comment:
It should be part of https://github.com/datarockets/ruby-style/blob/master/RELEASING.md